src/color-modifiers/modifyColor.ts

```diff
@@ -36,27 +39,41 @@ export function modifyColor(
   }
 
   baseColor = parseUIColor(baseColor);
-
   const color = new Color(baseColor);
   let returnedColor = color;
+
+  let resolvedMathFractionDigits: number = defaultFractionDigits;
+  if (modifier?.mathFractionDigits) {
+    resolvedMathFractionDigits = modifier.mathFractionDigits;
+  }
```

Comment:
```suggestion
  const resolvedMathFractionDigits = modifier.mathFractionDigits ?? defaultFractionDigits;
```
If coverage complains then I think we need to add a test to cover the branch that is not covered.
If typescript complains because modifier can be undefined then we need to fix that because it can't be undefined. I'd be really surprised if it thinks it can be, it's not an optional parameter. I can look into it more deeply if it's an issue.